tests/sp1/compile/basic/Cargo.toml

```diff
@@ -6,4 +6,4 @@ edition = "2021"
 [workspace]
```

Comment:
These SP1 example I pinned to tagged releases, since they were making the Docker builds fail, since this was pulling the default branch and the Docker downloads the latest tool tag and they weren't compatible.

Now everything points to the same tag, and the Docker build got fixed.